cloudslang-cli/src/main/java/io/cloudslang/lang/cli/SlangCLI.java

```diff
@@ -63,14 +63,15 @@
 
     @CliCommand(value = "run", help = "triggers a CloudSlang flow")
     public String run(
-            @CliOption(key = {"", "f", "file"}, mandatory = true, help = "Path to filename. e.g. cslang run --f C:\\CloudSlang\\flow.yaml") final File file,
-            @CliOption(key = {"cp", "classpath"}, mandatory = false, help = "Classpath , a directory comma separated list to flow dependencies, by default it will take flow file dir") final List<String> classPath,
+            @CliOption(key = {"", "f", "file"}, mandatory = true, help = "path to filename. e.g. cslang run --f C:\\CloudSlang\\flow.yaml") final File file,
```

Comment:
I just lowercase-ed the first word to be consistent with the other options. Should I change it back?